cpp/client/async_log_client.cc

```diff
@@ -464,12 +464,13 @@ void AsyncLogClient::InternalAddChain(const CertChain& cert_chain,
   jsend.Add("chain", jchain);
 
   UrlFetcher::Request req(GetURL(pre_cert ? "add-pre-chain" : "add-chain"));
+  req.verb = UrlFetcher::Verb::POST;
```

Comment:
I wondered the same, but it seems to me that once you start using other verbs, you'll also want to set a body, maybe some headers, and so on... So I went toward keeping this closer to a "pure ```struct```" (even though it's not all that "pure"!).